backend/src/handlers/dashboard.go

```diff
@@ -97,40 +97,44 @@ func (srv *Server) handleLoginMetrics(w http.ResponseWriter, r *http.Request, lo
 		if err != nil {
 			return newDatabaseServiceError(err)
 		}
-		totalUsers, err := srv.Db.GetTotalUsers(facilityId)
+		totalStudents, totalAdmins, err := srv.Db.GetTotalUsers(facilityId)
 		if err != nil {
 			return newDatabaseServiceError(err)
 		}
-		newAdded, err := srv.Db.NewUsersInTimePeriod(days, facilityId)
+		newUsersAdded, newAdminsAdded, err := srv.Db.NewUsersInTimePeriod(days, facilityId)
 		if err != nil {
 			return newDatabaseServiceError(err)
 		}
 		loginTimes, err := srv.Db.GetLoginActivity(days, facilityId)
 		if err != nil {
 			return newDatabaseServiceError(err)
 		}
-		if totalUsers == 0 {
-			totalUsers = 1
+		if totalStudents+totalAdmins == 0 {
+			totalStudents = 1
 		}
 		activityMetrics := struct {
 			ActiveUsers      int64                  `json:"active_users"`
 			TotalLogins      int64                  `json:"total_logins"`
 			LoginsPerDay     int64                  `json:"logins_per_day"`
 			PercentActive    int64                  `json:"percent_active"`
 			PercentInactive  int64                  `json:"percent_inactive"`
-			TotalUsers       int64                  `json:"total_users"`
+			TotalStudents    int64                  `json:"total_students"`
```

Comment:
Can we change this to be `TotalResidents` and `total_residents` now? It'll need an update in `common.ts` if we do but it appears the only areas it affects is the OperationalInsightsCharts.